content/20.treatments.md

```diff
@@ -518,28 +455,22 @@ The trial is currently recruiting and is due to run until March 2021.
-We don't yet understand if these findings apply to COVID-19.
+Thus, some evidence suggests that vitamin C supplementation or infusion can shorten the duration of a cold, reduce an individual's susceptibility to infections, and shorten a patient's stay in an ICU when administered at high doses, but we don't yet understand if these findings apply to COVID-19.
 There are ongoing trials in China and Italy that will inform our understanding of the therapeutic value of vitamin C supplementation for COVID-19.
-
+Once again, vitamin C intake is likely to be part of a healthy diet and the vitamin likely presents minimal risk, but its potential prophylactic or therapeutic effects against COVID-19 are unknown.
 
 ##### Vitamin D
 
 In terms of other dietary supplements, vitamin D can modulate the adaptive and innate immune system and has been associated with various aspects of health.
 Vitamin D can be sourced through diet or supplementation, but it is mainly biosynthesized by the body on exposure to sunlight.
-
-**Potential Mechanisms.**
 Vitamin D deficiency is associated with an increased susceptibility to infection [@doi:10.2310/JIM.0b013e31821b8755].
 In particular, vitamin D deficient patients are at risk of developing acute respiratory infections [@doi:10.1016/j.jsbmb.2012.11.017] and ARDS [@doi:10.1016/j.jsbmb.2012.11.017].
 1,25-dihydroxyvitamin D3 (25-hydroxyvitamin D) is the active form of vitamin D that is involved in adaptive and innate responses, whereby the vitamin D receptor is expressed in various immune cells and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes [@doi:10.2310/JIM.0b013e31821b8755; @doi:10.1016/j.coph.2010.04.001].
 Due to its potential immunomodulating properties, vitamin D supplementation may be advantageous to maintain a healthy immune system.
 
-**Current Evidence.**
 A recent review postulated that an individual’s vitamin D status may significantly affect their risk of developing COVID-19 [@doi:10.3390/nu12040988].
 This hypothesis was derived from the fact that the current pandemic emerged in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low.
-The authors suggest that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
+The authors suggested that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
 The authors also suggest high-dose supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff [@doi:10.3390/nu12040988].
```

Comment:
I will add one or two new studies here this evening, as vit D is probably the most promising with retrospective results already out